### PR TITLE
Move ipython into setup.py b/c it is not a dev tool

### PR DESCRIPTION
**Pull request by HiromuHota**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -30,6 +30,7 @@
         "tqdm>=4.26.0, <5.0.0",
         "treedlib>=0.1.1, <0.2.0",
         "wand>=0.4.4, <0.5.0",
+        "ipython",
```
Comment (SenWu) [inline setup.py]: can we have the version requirement as well?

Comment (HiromuHota) [reply setup.py]: At this moment, I have no idea about the boundary between those work and those don't.
But, I don't think it is a good idea to pose a strict version constraint on ipython like > 7.5.0 (latest).
Maybe `ipython>=7.0.0` is a good compromise?

Comment (HiromuHota) [reply setup.py]: Hold on. `ipython>=7.0.0` might give a wrong message to users that Fonduer should work with ipython of 7.0.0 or greater, which might not be the case.
I'd go with without a version constraint until we find the correct one.

Comment (SenWu) [reply setup.py]: Make sense!

**Pull request merged**